src/t8_element.c

```diff
@@ -55,10 +55,10 @@ const double
         { 1, 1, 1 } },
       { /* T8_ECLASS_PRISM */
         { 0, 0, 0 },
```

Comment:
This look-up-table is undocumented. Please add documentation

Reply:
done